### ISSUE TITLE
A live feedback system built into the app as a chat room

### ISSUE THREAD

**Issue by dennyabrain** [tattle-made/khoj-android#3]
<img width="499" alt="Screen Shot 2020-06-06 at 7 02 54 AM" src="https://user-images.githubusercontent.com/1415361/83933166-d4ddfa00-a7c3-11ea-8bd4-7a8355f7301a.png">
 
For engaging the initial userbase for the app (the first 1000 users), I want to experiment with having some form of a chatroom inside the app where we could ask them questions to understand where the app was falling short on their expectations. I am hoping it will also open them up to giving us more feedback and ideas.

the image on the right hand side, shows a proposed way of validating a user's phone number. My justification for it was that it would help us assign a phone number to the user. Which can be useful for blocking/moderating the person if they abuse the chat feature.

In our discussion with @tarunima and @kruttikanadig, it was decided that for the first 1000 users we get, we can do without phone number verification. that is, we will assume that abuse, spam and misuse of the chat feature won't be a big issue at this early stage.

Note : this feature is still tentative and not finalized whether we'll implement it or not. I'm just documenting this to open up discussions and see how everyone else feels about it. Any other suggestions to "get continuous feedback from users" are appreciated as well.

**Comment by harman25** [comment]
I am not too sure if chat will be very scalable, as users expect quick if not immediate response to chat messages they will post. 
In my opinion we can show the users a feedback screen after every (n) searches the user makes to get feedback on his/her experience with the app. We can have a timeline (I guess i discussed this with you during one of our calls) where the user can come back and give feedback on any particular search he chooses to.

**Comment by dennyabrain** [comment]
yes. i like your idea!  i agree with what you said about people's expectation with chat. i'd also been thinking of having an option of 'feedback' for each request they submit to us. Will have that reflect in the future mocks.
